ffcx/element_interface.py

```diff
@@ -25,30 +25,37 @@
 }
 
 
-def create_element(ufl_element):
+def create_element(ufl_element, *args):
```

Comment:
I've always had a dislike of `*args` for readability reasons. Is there a simple way around using it? If no, we need a detailed docstring.